atom/app/atom_main.cc

```diff
@@ -43,9 +43,9 @@
 #include "base/i18n/icu_util.h"
 #include "electron/buildflags/buildflags.h"
 
-#if defined(HELPER_EXECUTABLE)
+#if defined(ENABLE_SEATBELT)
```

Comment:
I'd rather the `HELPER_EXECUTABLE` code stay as-is and we change these `#ifs` to `#if defined(HELPER_EXECUTABLE) && !defined(MAS_BUILD)`. `ENABLE_SEATBELT` is a little misleading here because it will be undefined in the main executable even when seatbelt is enabled.

Reply:
I actually did it like that initially :) will revert to that version